src/types/BalanceDelta.sol

```diff
@@ -1,48 +1,67 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
+import {SafeCast} from "../libraries/SafeCast.sol";
+
 type BalanceDelta is int256;
```

Comment:
would you mind putting a comment above BalanceDelta describing that BalanceDelta is 2 int128s packed together, one for 0 and one for 1?
I just think with these assembly changes, its becoming less obvious at a quick glance what this type is haha

Reply:
Added.